### ISSUE TITLE
Change copy/move-results to use HTTPS instead of SSH

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#1024]
Instead of using SSH we should write a small Bottle or Flask python application deployed behind Gunicorn to accept a PUT of a file to a URL.

Once we have the MD5 value of the tarball, something like:
```
$ curl -X PUT http://pbench-server.example.com/upload/<MD5> -d @tarball.tar.xz
```

Would accept the tar ball file (max 10 GB to start), verify the tar ball is a proper `xz` compressed tar ball, that its computed MD5 checksum matches the `<MD5>` URL value, and then move it into place for `pbench-dispatch` processing.

**Comment by portante** [comment]
For satellite servers, we'd also write a GET method to serve tar balls that need to be pulled down, and a DELETE method that would remove them.

**Comment by portante** [comment]
Implemented by #1505.
